pkgs/development/libraries/rocclr/default.nix

```diff
@@ -16,28 +15,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-tYFoGafOsJYnRQaOLAaFix6tPD0QPTidOtOicPxP2Vk=";
   };
 
-  patches = [
-    # Enable support for gfx8 again
-    # See the upstream issue: https://github.com/RadeonOpenCompute/ROCm/issues/1659
-    # And the arch patch: https://github.com/rocm-arch/rocm-arch/pull/742
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/John-Gee/rocm-arch/d6812d308fee3caf2b6bb01b4d19fe03a6a0e3bd/rocm-opencl-runtime/enable-gfx800.patch";
-      hash = "sha256-59jFDIIsTTZcNns9RyMVWPRUggn/bSlAGrky4quu8B4=";
-    })
-  ];
```

Comment:
I think I just forgot to re-add it when rewritng.
This is something we should consider though, since it's been officially unsupported: https://github.com/RadeonOpenCompute/ROCm/issues/1353
Options:
- Remove entirely and if the user wants it they can override the derivation's attrs
- Keep but guard with a bool (enableGFX8?)
- Keep

I don't really think we should be adding this patch by default at least.

Reply:
Not having gfx8 support by default means that users that want it need to rebuild most of the rocm packages, because they won’t be in the binary cache.

Reply:
Fair enough.

Reply:
Thank you for catching this @Flakebi! I think it's okay (and very user friendly) to keep this patch by default since it just stops the code from [refusing](https://github.com/ROCm-Developer-Tools/ROCclr/blob/fcea5de094ef31525970b207d8e4e0e31de9610e/device/device.hpp#L1363) to run; it doesn't affect functionality on supported hardware.